### PR TITLE
Rephrasing items in changelog.txt

### PR DESCRIPTION
**Pull request by zaki50**
@realm/java please review this.

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
- * IllegalStateException is now thrown instead of ArrayIndexOutOfBoundsException if following methods in RealmList are called after closing the Realm or removing parent object: set(int,RealmObject), move(int,int), remove(int), get(int)
- * IllegalStateException is now thrown instead of returning 0/null if following methods in RealmList are called after closing the Realm or removing parent object: clear(), removeAll(Collection), remove(RealmObject), first(), last(), size(), where()
+ * Added RealmQuery.isValid(), RealmResults.isValid() and RealmList.isValid(). Each method checks whether the instance is still valid to use or not(for example, the Realm has been closed or any parent object has been removed).
+ * Following methods in RealmList now throw IllegalStateException instead of native crash when RealmList.isValid() returns false: add(int,RealmObject), add(RealmObject),
+ * Following methods in RealmList now throw IllegalStateException instead of ArrayIndexOutOfBoundsException when RealmList.isValid() returns false: set(int,RealmObject), move(int,int), remove(int), get(int)
```
Comment (cmelchior) [inline changelog.txt]: This is only true for managed RealmLists. un-managed RealmLists don't throw these exceptions

File: changelog.txt [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+ * Following methods in RealmList now throw IllegalStateException instead of returning 0/null when RealmList.isValid() returns false: clear(), removeAll(Collection), remove(RealmObject), first(), last(), size(), where()
```
Comment (cmelchior) [inline changelog.txt]: Same

File: changelog.txt (hunk 1 above)
Comment (zaki50) [inline changelog.txt]: @cmelchior good catch! I confined it to managed RealmLists.

**Comment by dalinaum** [comment]
:+1: It's in the middle of the night @zaki50  :)

**Comment by cmelchior** [comment]
:+1:

**Pull request merged**